server/src/main/java/io/druid/client/cache/CacheMonitor.java

```diff
@@ -22,15 +22,16 @@
 import com.metamx.emitter.service.ServiceMetricEvent;
 import com.metamx.metrics.AbstractMonitor;
 
-/**
- */
 public class CacheMonitor extends AbstractMonitor
 {
-  private final Cache cache;
+  volatile Cache cache;
```

Comment:
i guess `private` is removed to make it accessible inside unit test, can you put that as comment or may be expose a getCache() method?